Formula/aws-iam-authenticator.rb

```diff
@@ -0,0 +1,37 @@
+class AwsIamAuthenticator < Formula
+  desc "Use AWS IAM credentials to authenticate to Kubernetes"
+  homepage "https://github.com/kubernetes-sigs/aws-iam-authenticator"
+  url "https://github.com/kubernetes-sigs/aws-iam-authenticator/archive/v0.3.0.tar.gz"
+  sha256 "91e801c1b1c097f663ebd75897cd5ea19e709bc5ab806e1169ffc508c5a79271"
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    (buildpath/"src/github.com/kubernetes-sigs/aws-iam-authenticator").install buildpath.children
+
+    # this works around an upstream bug
+    # already fixed, not yet released
+    # see https://github.com/kubernetes-sigs/aws-iam-authenticator/commit/678cdff0f43d3ae4a1d9d68a5347042f14a168a8
+    mkdir buildpath/"src/github.com/heptio"
+    ln_s buildpath/"src/github.com/kubernetes-sigs/aws-iam-authenticator", buildpath/"src/github.com/heptio/authenticator"
```

Comment:
@Moisan @SMillerDev should we use [install_symlink](https://docs.brew.sh/Formula-Cookbook#file-level-operations) here or it's not needed during build phase?

Reply:
@igas if i understand correctly, `install_symlink` is for use when we want a symlink to exist in the final package.  in this case the symlink is temporary and will, i believe, be cleaned up after the build is finished.